tracer/test/snapshots/AwsSnsTests.NetFramework.SchemaV0.verified.txt

```diff
@@ -0,0 +1,82 @@
+[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: http.request,
```

Comment:
Side note: The two http.request spans in the snapshots correspond to additional SNS client requests (CreateTopicAsync and DeleteTopicAsync). Rather than block the PR on instrumenting these calls, I suggested that this be done as a followup PR.